src/lib/translations/de.ts

```diff
@@ -28,6 +28,7 @@ export const de: Translations = {
 	'settings.about.label': 'Du willst mehr wissen?',
 	'settings.about.button': 'Über Werwapp',
 	'settings.reset': 'Spiel zurücksetzten',
+	'settings.wakelock': 'Deaktiviere die automatische Bildschirmsperre',
```

Comment:
In general it´s working on my pc. However, i would recommend changing the label to "Aktiviere die automatische Bildschirmsperre",  as I thaught enabling the checkbox would disable the the wakelock instead of enabling it